src/surabhi/assignment15_16/BankAccount.java

```diff
@@ -0,0 +1,162 @@
+package surabhi.assignment15_16;
+
+import java.util.Scanner;
+
+public class BankAccount {
+
+
+	int currentICBal, currentHDFCbal, currentBOIbal, currentSBIbal;
+	int debitHDFC, creditHDFC, debitSBI, creditSBI, debitBOI, creditBOI, debitIC, creditIC;
+	int debitAmtCount, creditAmtCount;
+	
+	void setInitialBalance(int balICICI, int balHDFC, int balSBI, int balBOI) {
+		currentICBal=balICICI;
+		currentHDFCbal=balHDFC;
+		currentBOIbal=balBOI;
+		currentSBIbal=balSBI;
+	}
+	
+	void processDetails(String bankName, String operation, int amount) {
+				switch(bankName) {
+					case "hdfc":
+						{
+							switch(operation) {
+						case "debit":
+							if(currentHDFCbal>=amount) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.